src/pages/index.tsx

```diff
@@ -59,7 +59,8 @@ const Home: NextPage = () => {
       <footer>
         <div className="flex justify-center">
           <p className="mb-[20px] text-center text-base leading-6 text-neutral-light md:mb-[70px] md:text-left">
-            Web3 Philippines &amp; Wareneutron Developers
+            <strong>Directory v1.2.0</strong> by Web3 Philippines &amp;
```

Comment:
we probably don't need to make the version number to be bold here. a shade same with the last updated text should be fine here.

Reply:
got it thanks for the input Ivan!